.github/workflows/check-dist-sha256.yml

```diff
@@ -0,0 +1,51 @@
+name: Check dist sha256
+# Validates whether the dist folder is as expected.
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+  workflow_dispatch:
+
+jobs:
+  check-dist:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+
+      - name: Set Node.js 16
+        uses: actions/setup-node@2fddd8803e2f5c9604345a0b591c3020ee971a93
+        with:
+          node-version: 16
+
+      - name: Install dependencies
+        run: |
+          cd .github/actions/compute-sha256
+          npm i
+
+      - name: Rebuild the dist/ directory
+        run: |
+          cd .github/actions/compute-sha256
+          npm run all
+
+      - name: Compare the expected and actual dist/ directories
+        run: |
+          if [ "$(git diff --ignore-space-at-eol dist/ | wc -l)" -gt "0" ]; then
```

Comment:
for some reason I'm not able to verify locally. git diff gives me a bunch of diff:
```diff
--- a/.github/actions/compute-sha256/dist/index.js
+++ b/.github/actions/compute-sha256/dist/index.js
@@ -43,16 +43,16 @@ function shasum256(untrustedPath) {
         throw new Error(`File ${untrustedPath} not present`);
     }
     const untrustedFile = fs.readFileSync(untrustedPath);
-    return crypto.createHash('sha256').update(untrustedFile).digest('hex');
+    return crypto.createHash("sha256").update(untrustedFile).digest("hex");
 }
 function run() {
     return __awaiter(this, void 0, void 0, function* () {
         // Get the path to the untrusted file from ENV variable 'UNTRUSTED_PATH'
-        const untrustedPath = core.getInput('untrusted_path');
+        const untrustedPath = core.getInput("untrusted_path");
         core.info(`Computing sha256 of ${untrustedPath}`);
         const sha = shasum256(untrustedPath);
         core.info(`Computed sha256 of ${untrustedPath} as ${sha}`);
-        core.setOutput('sha256', sha);
+        core.setOutput("sha256", sha);
     });
 }
 run();

diff --git a/.github/actions/compute-sha256/dist/index.js.map b/.github/actions/compute-sha256/dist/index.js.map
index 019ee85..fc96d7c 100644
--- a/.github/actions/compute-sha256/dist/index.js.map
+++ b/.github/actions/compute-sha256/dist/index.js.map
@@ -1 +1 @@
-{"version":3,"file":"index.js","sources":["../webpack://typescript-action/./lib/main.js","../webpack://typescript-action/./node_modules/@actions/core/lib/command.js","../webpack://typescript-action/./node_modules/@actions/core/lib/core.js","../webpack://typescript-action/./node_modules/@actions/core/lib/file-command.js","../webpack://typescript-action/./node_modules/@actions/core/lib/oidc-utils.js","../webpack://typescript-action/./node_modules/@actions/core/lib/utils.js","../webpack://typescript-action/./node_modules/@actions/http-client/auth.js","../webpack://typescript-action/./node_modules/@actions/http-client/index.js","../webpack://typescript-action/./node_modules/@actions/http-client/proxy.js","../webpack://typescript-action/./node_modules/tunnel/index.js","../webpack://typescript-action/./node_modules/tunnel/lib/tunnel.js","../webpack://types [...]
```

What does npm version give you? Mine is:
```
{
  'typescript-action': '0.0.0',
  npm: '8.10.0',
  node: '14.19.1',
  v8: '8.4.371.23-node.85',
  uv: '1.44.1',
  zlib: '1.2.11',
  brotli: '1.0.9',
  ares: '1.18.1',
  modules: '83',
  nghttp2: '1.43.0',
  napi: '8',
  llhttp: '2.1.4',
  openssl: '1.1.1n',
  cldr: '37.0',
  icu: '67.1',
  tz: '2019c',
  unicode: '13.0'
}
```

Reply:
I think this is because 'npm run all' runs 'npm run format' formats the output using 'prettier'. @naveensrinivasan needs to run 'npm run format' and commit the changed files.

Reply:
maybe also format before building, even if it makes not difference in practice.

Reply:
I have updated it. thanks